mlflow/utils/databricks_utils.py

```diff
@@ -2,13 +2,20 @@
 import logging
 import os
 import subprocess
+from sys import stderr
 from typing import Optional, TypeVar
 
-from databricks_cli.configure import provider
-
 import mlflow.utils
 from mlflow.environment_variables import MLFLOW_TRACKING_URI
 from mlflow.exceptions import MlflowException
+from mlflow.legacy_databricks_cli.configure.provider import (
+    DatabricksConfig,
+    DatabricksConfigProvider,
+    DefaultConfigProvider,
+    ProfileConfigProvider,
+    get_config,
+    set_config_provider,
+)
```

Comment:
we can import provider.

Reply:
I prefer to directly importing names from provider module, the `provider` module name does not have very clear meaning.